src/doNothing.ts

```diff
@@ -20,15 +20,15 @@
  * @dataLast
  * @category Function
  */
-export function doNothing(): (
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ...args: any
-) => void {
-  // Because the function does nothing we safely return the same function
-  // (pointer) and don't need to create a new one on every invocation.
-  return doesNothing;
-}
+export const doNothing = (): typeof doesNothing =>
```

Comment:
**question** if we do this, then the hover for `doNothing` will show `(): typeof doesNothing` rather than `(): <Args extends ReadonlyArray<unknown>>(...args: Args) => void` right?

think this is less useful, so i think it's better to repeat the type here...

Reply:
It looks like VSCode is doing a good job of dereferencing for these sort of things. It most likely not even vscode doing it, but typescript itself, so it's probably the same in IntelliJ

![image](https://github.com/remeda/remeda/assets/4711558/14e5e396-84bd-4ce3-86aa-70b4b1e446ed)

Reply:
ah, cool. yes, it it typescript itself (it's tsserver), so it's the same across editors